drivers/sensor/lps22hh/lps22hh.c

```diff
@@ -207,38 +202,92 @@ static int lps22hh_init(const struct device *dev)
 	return 0;
 }
 
-static struct lps22hh_data lps22hh_data;
+#if DT_NUM_INST_STATUS_OKAY(DT_DRV_COMPAT) == 0
+#warning "LPS22HH driver enabled without any devices"
+#endif
+
+/*
+ * Device creation macro, shared by LPS22HH_DEFINE_SPI() and
+ * LPS22HH_DEFINE_I2C().
+ */
```

Comment:
I can't see `LPS22HH_DEFINE_SPI` nor `LPS22HH_DEFINE_I2C`, this macro seems to be used by `LPS22HH_DEFINE` only (so `DEVICE_DT_INST_DEFINE` may be placed there instead)

Reply:
agree